app/forms/work_form.rb

```diff
@@ -19,18 +19,18 @@ class WorkForm < Reform::Form
   property :license
   property :agree_to_terms
 
-  property 'created(1i)', virtual: true
-  property 'created(2i)', virtual: true
-  property 'created(3i)', virtual: true
-  property 'created_range(1i)', virtual: true
-  property 'created_range(2i)', virtual: true
-  property 'created_range(3i)', virtual: true
-  property 'created_range(4i)', virtual: true
-  property 'created_range(5i)', virtual: true
-  property 'created_range(6i)', virtual: true
-  property 'published(1i)', virtual: true
-  property 'published(2i)', virtual: true
-  property 'published(3i)', virtual: true
+  property 'created(1i)', virtual: true, type: ::Types::Custom::NilableInteger
```

Comment:
See comment in `config/initializers/dry_types.rb` as to what this type does and why I defined it.